### PR TITLE
fix for BSD's sed

### PR DESCRIPTION
**Pull request by PDoakORNL**
While fixing #695 This leaves OSX able to get git rev and I assume other BSD's as well. It works with  cmake 2.8.10.2 on titan.

### CODE REVIEW

**Comment by prckent** [comment]
OK to test

**Comment by prckent** [comment]
I think it would be more robust to check for -E and/or -r support and then use the appropriate flags. A hardwired list is less future proof and for sure there are non-GNU variants not there.

**Comment by PDoakORNL** [comment]
Ok this should last until sed -r only machines all die. Although I'm sure there is a way to implement that custom command without extended regexp in sed if someone wants to go for it.

File: manual/installation.tex [hunk 1]
```diff
@@ -614,7 +614,7 @@ \subsection{Installing on ORNL OLCF Titan Cray XK7 (NVIDIA GPU
 module load cray-hdf5
 module load cmake
 module load fftw
-export FFTW_HOME=$FFTW_DIR/..
+export FFTW_HOME=$(echo $FFTW_DIR | sed -e 's/\/lib//')
```
Comment (ye-luo) [inline manual/installation.tex]: Please explain this change. Is this causing problems? I used the build script yesterday on titan and it was fine.

**Pull request merged**